.env.template.mainnet.eon

```diff
@@ -3,8 +3,25 @@ COMPOSE_PROJECT_NAME=evmapp-mainnet-eon
 INTERNAL_NETWORK_SUBNET='10.10.50.0/24'
 SCNODE_IP_ADDRESS='10.10.50.3'
 
+ZEN_TAG=v4.1.0
+ZEN_PORT=19033
+ZEN_RPC_USER=zenuser
+ZEN_RPC_PASSWORD=password
+ZEN_RPC_PORT=8231
+ZEN_WS_PORT=8888
+ZEN_RPC_ALLOWIP_PRESET=SUBNET
+ZEN_EXTERNAL_IP=DETECT
+ZEN_OPTS="-websocket=1 -wsaddress=0.0.0.0 -wsport=8888 -logips=1 -logtimestamps=1 -disabledeprecation=4.1.0 -debug=ws -debug=sc -debug=cert"
+ZEN_CUSTOM_SCRIPT=""
+ZEN_SEED_TAR_GZ_URL=""
+ZEN_LOG=STDOUT
+ZEN_LOCAL_USER_ID=999
+ZEN_LOCAL_GRP_ID=999
+
 # Sidechain config
-SCNODE_FORGER_ENABLED=false
+SCNODE_ROLE=
+SCNODE_FORGER_ENABLED=
+SCNODE_FORGER_MAXCONNECTIONS=20
```

Comment:
To be honest I would prefer to set it to the default (20) without prompting for the value. But we can populate it only if the node is a forger even if essentially nothing will change (all the nodes just default to 20 when the parameter is not provided). I kept the variable here because it could anyway be useful to have the possibility to manually adjust.